MachineLearningPrototype/dataCollection/collect_data.py

```diff
@@ -119,30 +120,27 @@ def fetch_github_issues(args, db):
 
 			#Get the last updated time for the repo
 			since_info = get_collection_record({'repository': repo}, None, 'LastUpdatedInfo', db)
-
 			num_issues = 0
 			since = None
-
-			#Get the last_updated time and total_issues_collected if the repo is found
+			
```

Comment:
Also here. Thanks.

Reply:
Can you please check if it is fine now? Thank you

Reply:
The first one got fixed, but this one still remains https://github.com/adoptium/aqa-test-tools/pull/455/files

Reply:
Couldn't remove those extra spaces with that comment, so removed the comment :sweat_smile: